job_definitions/generate_upload_and_notify_counterpart_signature_pages.yml

```diff
@@ -96,6 +97,34 @@
             ]
           }
       }
+  
+- job:
+    name: 'generate-upload-and-notify-counterpart-signature-pages-periodic'
+    display-name: 'Generate, upload and notify counterpart signature pages for {{', '.join(s) for s,_,enabled if enabled}}'
```

Comment:
Can we include the `<stage>` in the `display-name` if it's not a parameter we can set when building the job?